docs/web-apps/automated-testing/cypress/yaml/v1alpha.md

```diff
@@ -0,0 +1,898 @@
+
+---
+
+### `name`
+<p><small>| REQUIRED | STRING |</small></p>
+
+The name of the test suite, which will be reflected in the results and related artifacts.
+
+```yaml
+  - name: "saucy test"
+```
+---
+
+### `browser`
+<p><small>| REQUIRED | STRING |</small></p>
+
+The name of the browser in which to run this test suite.
+Available browser names: `chrome`, `firefox`, `microsoftedge`(only for sauce mode) and `electron`(only for docker mode).
+
+```yaml
+    browser: "chrome"
+```
+---
+
+### `browserVersion`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The version of the browser to use for this test suite.
+
+```yaml
+    browserVersion: "85.0"
+```
+---
+
+### `platformName`
+<p><small>| OPTIONAL | STRING | <span class="highlight sauce-cloud">Sauce Cloud only</span> |</small></p>
+
+A specific operating system and version on which to run the specified browser and test suite. Defaults to a platform that is supported by `saucectl` for the chosen browser.
+
+```yaml
+    platformName: "Windows 10"
+```
+---
+
+### `screenResolution`
+<p><small>| OPTIONAL | STRING | <span class="highlight sauce-cloud">Sauce Cloud only</span> |</small></p>
+
+Specifies a browser window screen resolution, which may be useful if you are attempting to simulate a browser on a particular device type. See [Test Configurations](/basics/test-config-annotation/test-config) for a list of available resolution values.
+
+```yaml
+    screenResolution: "1920x1080"
+```
+---
+
+### `mode`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies whether the individual suite will run on `docker` or `sauce`, potentially overriding the default project mode setting.
+
+```yaml
+  mode: "sauce"
+```
+---
+
+### `config`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Provides details related to the Cypress test configuration that are relevant for this test suite.
+
+```yaml {5}
+  suites:
+    - name: "Hello"
+      browser: "firefox"
+      platformName: "Windows 10"
+      config:
+        env:
+          hello: world
+        testFiles: [ "**/*.spec.js" ]
+```
+---
+
+#### `env`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+A property containing one or more environment variables that may be referenced in the tests for this suite. Expanded environment variables are supported. Values set here will be overwritten by values set in the global `env` property.
+
+```yaml
+  config:
+    env:
+      hello: world
+      my_var: $MY_VAR
+```
+
+:::caution
+Since environment variables are provided to Cypress directly, avoid using `CYPRESS_` as a prefix.
+:::
+
+---
+
+#### `testFiles`
+<p><small>| REQUIRED | STRING/ARRAY/REGEX |</small></p>
+
+One or more paths to the Cypress test files to run for this suite, if not otherwise specified explicitly in `cypress.json`. Regex values are supported to indicate all files of a certain type or in a certain directory, etc.
```

Comment:
hmm.. also should be glob pattern for `testFiles` and `excludedTestFiles` https://docs.cypress.io/guides/references/legacy-configuration#Folders-Files